cli/command/container/create.go

```diff
@@ -242,7 +243,7 @@ func createContainer(ctx context.Context, dockerCli command.Cli, containerCfg *c
 			return err
 		}
 		if taggedRef, ok := namedRef.(reference.NamedTagged); ok && trustedRef != nil {
-			return image.TagTrusted(ctx, dockerCli, trustedRef, taggedRef)
+			return trust.TagTrusted(ctx, dockerCli.Client(), dockerCli.Err(), trustedRef, taggedRef)
 		}
```

Comment:
Looks like this introduced a regression; most likely because before this `dockerCli.Client()` would be initialised lazily, and it's possible some settings were loaded elsewhere.

I opened a PR that reverts some of the changes;

- https://github.com/docker/cli/pull/5905

Reply:
Looks like reverting those changes (PR above) still makes things fail occasionally.

I think it's just a bad test and/or multiple tests affecting each other (perhaps using same port or otherwise)